### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by GovindSharma0708**
added loader and fixed header page review and please merge pr

### CODE REVIEW

**Comment by GovindSharma0708** [comment]
@manzil-infinity180 pr  raised please merge it.

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -8,7 +8,11 @@ function Header() {
             marginTop:"250px",
             backgroundColor:"#1f1e1e55"
         }}>
-             <h1 className='title_header'>Your Review/Suggestion for this Project</h1>
+             
+             
+            <h1 class="mb-4 font-extrabold text-gray-900 dark:text-white "><span class="text-transparent bg-clip-text bg-gradient-to-r to-emerald-600 from-sky-400">For XF </span>Your Feedback </h1>
```
Comment (manzil-infinity180) [inline src/components/Header/Header.jsx:13]: in. react there is no class replace with className and this code look like it is tailwind css

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -63,10 +67,34 @@ function Header() {
             alignItems: "center" /* Ensure they are vertically aligned */
 
          }}>
+          
+
+<footer class="bg-white rounded-lg shadow dark:bg-gray-900 m-4">
+    <div class="w-full max-w-screen-xl mx-auto p-4 md:py-8">
+        <div class="sm:flex sm:items-center sm:justify-between">
+
           <a href='https://github.com/manzil-infinity180' style={{textDecoration:"none",margin: "10px"}}><h1 className='h1_header'>About Me</h1></a>
```
Comment (manzil-infinity180) [inline src/components/Header/Header.jsx:76]: here too

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -63,10 +67,34 @@ function Header() {
...
           <a href="https://github.com/manzil-infinity180/Xf-Registration" style={{textDecoration:"none", margin: "10px"}}><h1 className='h1_header'>Contribute</h1></a>
            <a href="https://forms.gle/yz9WJEHkm7ahvu4o9" style={{textDecoration:"none", margin: "10px"}}><h1 className='h1_header'>Suggestion</h1></a>
+
            
+            <ul class="flex flex-wrap items-center mb-6 text-sm font-medium text-gray-500 sm:mb-0 dark:text-gray-400">
```
Comment (manzil-infinity180) [inline src/components/Header/Header.jsx:81]: change to normal css

**Review by manzil-infinity180 — CHANGES_REQUESTED** [review]
do. it and then add a small video or screenshot i will merge it

File: src/components/Header/Header.jsx (hunk 2 above)
Comment (GovindSharma0708) [reply src/components/Header/Header.jsx:76]: changes are done please review and merge it @manzil-infinity180

**Pull request closed without merging**